sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -431,6 +432,14 @@ def run_pipeline(self, pipeline, options):
       else:
         debug_options.add_experiment('use_staged_dataflow_worker_jar')
 
+    # Make Dataflow workers use FastAvro on Python 3 unless use_avro experiment
+    # is set. Note that use_avro is only interpreted by the Dataflow Runner
```

Comment:
Did you mean "only interpreted by the Dataflow SDK and not interpreted by Dataflow service/workers" ?

Reply:
Updated, thanks!